andela/ah-frontend-prime
# #166107267 Enable Users Edit and Delete their articles

### What does this PR do?

Enables users edit and delete only artilces they created.

#### Description of Task to be completed?

- Add page for users to edit articles they created
- Add functionality of deleting an article

#### How should this be manually tested?
Clone the repo

- `$ git clone https://github.com/andela/ah-frontend-prime.git`

Check into the project directory and Check out the branch  `ft-edit-delete-articles-166107267`

- `$ cd ah-frontend-prime`
- `$ git checkout  ft-edit-delete-articles-166107267`

Install all dependencies

- `$ npm install`

Start the server

- `$npm start`

Create an acount and login with the sign up and sign in buttons on the dashboard

Create an article By hovering on the picture in the right corner and clicking create article.
Add all necessary fields

View the articles you created by hovering over the profile picture and clicking `Your stories`

Click edit on whichever article you'd like to edit to be provided a form in which you'll enter all the necessary changes you'd like to make to edit the article.

Click delete to delete an article amongst your stories.


#### What are the relevant pivotal tracker stories?
[#166107267](https://www.pivotaltracker.com/story/show/166107267)

#### Screenshots (if appropriate)
Create Article

![Screenshot 2019-05-26 at 16 43 12](https://user-images.githubusercontent.com/43245704/58382705-3e82d000-7fd6-11e9-8003-75f3ded0f3f3.png)

User Articles

![Screenshot 2019-05-26 at 16 43 24](https://user-images.githubusercontent.com/43245704/58382709-3f1b6680-7fd6-11e9-8a0e-c2f47b0ba30f.png)

Edit Article Page

![Screenshot 2019-05-26 at 16 43 44](https://user-images.githubusercontent.com/43245704/58382704-3e82d000-7fd6-11e9-9da0-3a76deb6622b.png)

Delete Article

![Screenshot 2019-05-26 at 16 44 35](https://user-images.githubusercontent.com/43245704/58382707-3f1b6680-7fd6-11e9-907d-30660eaf8433.png)
